api/src/main/java/org/openmrs/scheduler/db/SchedulerDAO.java

```diff
@@ -39,6 +39,12 @@
 	 */
 	public TaskDefinition getTask(Integer taskId) throws DAOException;
 	
+	/**
+	 * @see org.openmrs.scheduler.SchedulerService#getTaskByUuid(String uuid)
```

Comment:
Can you use a full package for the String parameter?

Reply:
@dkayiwa i have added it